OpenRA.Game/TranslationProvider.cs

```diff
@@ -0,0 +1,62 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using OpenRA.FileSystem;
+
+namespace OpenRA
+{
+	public class TranslationProvider
```

Comment:
```suggestion
	public static class TranslationProvider
```
since everything about the class is static.